packages/app-layout/test/app-layout.test.js

```diff
@@ -261,7 +310,7 @@ describe('vaadin-app-layout', () => {
       // force non-overlay mode as default
       layout.style.setProperty('--vaadin-app-layout-drawer-overlay', 'false');
       layout._updateOverlayMode();
-      await nextFrame();
+      await nextRender();
```

Comment:
The app-layout delays some computations until the next render tick so it is better to wait for the next render here.